src/coord/src/catalog.rs

```diff
@@ -1069,14 +1079,7 @@ impl Catalog {
         }
     }
 
-    #[must_use]
-    pub fn insert_item(
-        &mut self,
-        id: GlobalId,
-        oid: u32,
-        name: FullName,
-        item: CatalogItem,
-    ) -> Event {
+    pub fn insert_item(&mut self, id: GlobalId, oid: u32, name: FullName, item: CatalogItem) {
```

Comment:
Did you mean to get rid of `#[must_use]`?

Reply:
Yeah it used to be there to prevent dropping the returned `Event` on the floor, but now this doesn't return an event, so nothing to use!